torch/csrc/jit/mobile/debug_info.cpp

```diff
@@ -140,7 +139,7 @@ MobileDebugTable::MobileDebugTable(
       }
 
       for (auto& val : lines.toTuple()->elements()) {
-        auto tup_elems = std::move(*std::move(val).toTuple()).elements();
```

Comment:
This is valid though? Is it used somewhere else in the loop? This calls the rvalue overload of toTuple if there are multiple implementations.

Reply:
val has type of const IValue&

Reply:
Oh, nvm then.